src/sentry/static/sentry/app/api.tsx

```diff
@@ -256,8 +295,8 @@ export class Client {
             errorObjectToUse.removeFrames(2);
 
             // Setting this to warning because we are going to capture all failed requests
-            scope.setLevel('warning');
-            scope.setTag('http.statusCode', resp.status);
+            scope.setLevel(Sentry.Severity.Warning);
+            scope.setTag('http.statusCode', String(resp.status));
```

Comment:
Unsure if this is a "bug".

Reply:
Don't think so. Tag values do need to be strings.

Reply:
We were actually sending it incorrectly before.

Reply:
Yea I'm converting it properly here now.